e2e/BlockUtils.e2e.ts

```diff
@@ -15,7 +15,7 @@ const networks: { [chainId: number]: string } = {
 describe("BlockUtils", () => {
   let rpcProviders: Provider[] = [];
 
-  beforeAll(() => {
+  beforeEach(() => {
```

Comment:
Resolved offline. Tl;dr I changed them to beforeEach so that the test cases pass. I think there's some level of non-determinism that I'm running into.